test/functional/cmdLineTests/gcRegressionTests/src/com/ibm/tests/garbagecollector/SpinAllocate.java

```diff
@@ -29,18 +29,25 @@
 public class SpinAllocate
 {
 	public static Object _objectHolder;
+	
+	/* Given that the jdk runs very slow on RISC-V, we have to increase 
+	 * the threshold value to ensure the test is completed.
+	 */
+	private static String archName = System.getProperty("os.arch");
+	private static boolean isRiscv = archName.toLowerCase().contains("riscv");
+	private static int threshold = (isRiscv) ? 200 : 60;
 
 	/**
 	 * @param args Takes one argument:  number of seconds to spin for before terminating with a message that the test ran to completion.
-	 * This argument is required.  It must be in the range [1-60]
+	 * This argument is required.  It must be in the range [1-200]
 	 */
 	public static void main(String[] args)
 	{
 		if (1 == args.length)
 		{
 			int secondsToSpin = Integer.parseInt(args[0]);
 
-			if ((secondsToSpin >= 1) && (secondsToSpin <= 60))
+			if ((secondsToSpin >= 1) && (secondsToSpin <= threshold))
```

Comment:
I believe you can just replace "60" to "200" regardless of platform. This extends valid range for parameter only.

Reply:
It makes more sense to me this way in terms of RISC-V.  Will update as long as it works for other platforms,  in which case the timeout setting on platforms might be adjusted accordingly.